### DISCUSSION TITLE
casaOS running on 10.0.0.22 instead of localhost/127.0.0.1

### DISCUSSION THREAD

**Discussion by bharathganji** [IceWhaleTech/CasaOS#1396] [Q&A]
hi team, 

i've installed casaOS on oracle free-tier ubuntu vps

[  OK  ] Your hardware architecture is : aarch64
[  OK  ] Your System is : Linux
[  OK  ] Your Linux Distribution is : ubuntu
[  OK  ] Memory capacity check passed.
[  OK  ] Disk capacity check passed.

 CasaOS v0.4.4.2 is running at:
 ─────────────────────────────────────────────────────
 - http://10.0.0.22 (enp0s6)
 Open your browser and visit the above address.

the only way i can access it is using public ip i've associated with vps.
but 10.0.022 is not my ip address(144.126.xx.xx). 

port 80  is open and when i type 144.126.xx.xx in browser, nothing is shown

please help me with steps to access via public IP.

edit: i've found 10.0.0.22 is my private ip attached to same vcn

**Comment by leesonaa** [comment]
try to use http://144.126.xx.xx  |  check firewall
